File: Web/src/Models/ProductModel.ts
export class ProductModel{
    id : number;
    description : string;
    code : string;
    price : number;
    grossWeight : number;
    netWeight : number;

    constructor() {
        this.id = 0;
        this.code = '';
        this.description = '';
        this.grossWeight = 0;
        this.netWeight = 0;
        this.price = 0;
    }
}

File: Api/Models/ProductModel.cs
using Api.Domain;

namespace Api.Models;

public class ProductModel : BaseModel
{
    public string Description { get; set; }

    public string Code { get; set; }

    public decimal Price { get; set; }

    public decimal GrossWeight { get; set; }

    public decimal NetWeight { get; set; }

    public ProductModel()
    {
        this.Description = string.Empty;
        this.Code = string.Empty;
    }
}